core/src/main/kotlin/com/katanox/tabour/consumption/Config.kt

```diff
@@ -0,0 +1,3 @@
+package com.katanox.tabour.consumption
+
+internal interface Config
```

Comment:
Question: Is this empty intentionally?

Reply:
yes, we need to indicate that classes are implementing an interface but that interface does not have any functions for now